db/schema.rb

```diff
@@ -11,7 +11,56 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20150701042203) do
+ActiveRecord::Schema.define(:version => 20160126231512) do
+
+  create_table "credence_answers", :force => true do |t|
+    t.integer  "credence_question_id"
```

Comment:
That's exactly it. Foreign keys are only SQL Integers in rails migrations. I'm not sure of the exact reason, but it might have something to do with compatibility across different types of SQL.

Awesome. Feel free to either add the index in the existing migrations that create the tables, or create new migrations that add them. Either works :)